### PR TITLE
[Storybook] General config and theme changes 

### PR DESCRIPTION
**Pull request by Gmantiqueira**
## What's the purpose of this pull request?

- Basic theme config with the VTEX brand color. In the future we can have a visual identity out of VTEX brand for our component library.
- Title `Storybook` changes to `StoreUI vX.X.X`.
- Adds a tab that shows the story code source, to see the entire implementation of the component.
- Sort the stories list in this order: `Getting Started` > `Atoms` > `Molecules` > `Organisms` > `Releases`
- Create a `Releases` section in our Storybook, with a Migration Guide and CHANGELOG templates.

### CODE REVIEW

**Comment by tlgimenes** [comment]
Somehow I'm getting the old storybook logo on the skeleton on the left when it's loading

**Comment by Gmantiqueira** [comment]
> Somehow I'm getting the old storybook logo on the skeleton on the left when it's loading

There's some cache or something that's annoying me. I'm trying to find out what's happening, thx.

**Comment by emersonlaurentino** [comment]
> Somehow I'm getting the old storybook logo on the skeleton on the left when it's loading

for me too

File: packages/store-ui/.storybook/Welcome/Welcome.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export const Welcome = () => {
+  return (
+    <div
+      className="welcome__container"
+      style={{
+        background: `url() no-repeat center center fixed`,
+        backgroundSize: 'cover',
+      }}
+    >
+      <h2 className="welcome__heading">Store Framework Jamstack</h2>
```
Comment (tlgimenes) [inline packages/store-ui/.storybook/Welcome/Welcome.js:14]:
```suggestion
      <h2 className="welcome__heading">Store UI</h2>
```

File: packages/store-ui/.storybook/Welcome/Welcome.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      <h4 className="welcome__heading welcome__heading--subtitle">{`Store UI v${PackageInfo.version}`}</h4>
```
Comment (tlgimenes) [inline packages/store-ui/.storybook/Welcome/Welcome.js:15]:
```suggestion
      <h4 className="welcome__heading welcome__heading--subtitle">{`Store UI v${PackageInfo.version}. A complete, framework agnostic library for building your next digital experience`}</h4>
```

Comment (emersonlaurentino) [reply packages/store-ui/.storybook/Welcome/Welcome.js:15]: Bold

**Review by tlgimenes — COMMENTED** [review]
What about this? Will this change once we make a release?
image

**Review by emersonlaurentino — APPROVED** [review]

**Comment by Gmantiqueira** [comment]
> What about this? Will this change once we make a release?
> image

Yeah, I put in there just for template and make it more easy when we start releasing stable versions. We can delete after the first version.

File: packages/store-ui/.storybook/Welcome/Welcome.js (hunk 2 above)
Comment (Gmantiqueira) [reply packages/store-ui/.storybook/Welcome/Welcome.js:15]: @emersonlaurentino Bold where?

**Review by tlgimenes — APPROVED** [review]
AHTM!

**Pull request merged**